src/decaygen.py

```diff
@@ -88,7 +89,7 @@
     switch (nucname::znum(it->first)) {
       {{ cases|indent(6) }}
       default:
-        outcomp.insert(*it);
+        outcomp.insert(std::make_pair(nucname::id_to_state_id(it->first), it->second));
```

Comment:
Actually, I don't think that this is right because outcomp is keyed by id, not state id.